logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/message/SendTextMessageUseCase.kt

```diff
@@ -103,16 +99,11 @@ class SendTextMessageUseCase internal constructor(
                 isSelfMessage = true
             )
             persistMessage(message)
-                .flatMap {
-                    messageSender.sendMessage(message)
-                }
-                .onSuccess {
-                    messageRepository.updateMessageStatus(MessageEntity.Status.SENT, conversationId, generatedMessageUuid)
```

Comment:
This is already done in `MessageSender` (`promoteMessageToSentUpdatingServerTime`)

Reply:
In case we wanted to for example, log that a self deleting message has been successfully sent (and trigger the timer for self user self deleting messages) this would be the place right?

If so maybe we could leave a comment or unify all these use cases "send successfully message event" with a common function? cc @trOnk12

Reply:
The thing is now we have it in one place for every use case that sends messages, it's in messageSender.sendMessage which uses messageRepository.promoteMessageToSentUpdatingServerTime. 🙂
Mateusz said that timer for self deleting messages is started and running on the UI so there's nothing do to in this regard in kalium and we would need to pass the information whether the message has been sent to the UIMessage which I did in this PR: https://github.com/wireapp/wire-android-reloaded/pull/1803